python/ray/autoscaler/_private/commands.py

```diff
@@ -416,9 +424,13 @@ def run_docker_stop(node, container_name):
 
     container_name = config.get("docker", {}).get("container_name")
     if container_name:
+        threads = []
```

Comment:
Why don't we use the thread pool instead of creating a thread per node? The user who required this feature will have 500 nodes.

Reply:
@rkooo567 , this is a great idea, why don't we create a thread pool of size say 500 and run all the stops in it?

Reply:
This will solve the previous issue if the number of nodes is too large and there can't be more threads.